cost_rules/app.py

```diff
@@ -1,4 +1,5 @@
 import json
+import logging
```

Comment:
you'll probably want to initialize the logger with these statements?

```
log = logging.getLogger(__name__)
log.setLevel(logging.INFO)
```

Reply:
I was going to rely on the default config, bug forgot that it only logs warning and above.